src/charm.py

```diff
@@ -187,28 +188,10 @@ def __init__(self, *args: Any) -> None:
             self._on_database_integration_broken,
         )
 
-        # admin ingress
-        self.framework.observe(self.admin_ingress.on.ready, self._on_admin_ingress_ready)
```

Comment:
Why is this removed? Don't we still need to observe the admin ingress-related events like we continue for the public one?

Reply:
this is because we don't do anything for admin-ingress integration joining and breaking.

Reply:
what I understand is that:

- We integrate with an admin ingress to expose admin API endpoints if we would like some client (outside the k8s cluster) to query
- For integration `hydra_endpoints`, we only pass the k8s svc addresses which are fixed values and for internal-cluster usage.

So admin ingress integration joining/departing does not change anything.

Let me know if I am wrong (we can chat a bit tomorrow)

Reply:
Ok, I missed this part:

> For integration hydra_endpoints, we only pass the k8s svc addresses which are fixed values and for internal-cluster usage.

It's fine to remove it then.